### DISCUSSION TITLE
SK6812/WS2812 missing slider on ESP32C3

### DISCUSSION THREAD

**Discussion by TheBattleWolf** [arendst/Tasmota#15227] [Q&A] [answered]
### PROBLEM DESCRIPTION
_After compiling, flashing and setting up Tasmota the Sliders for the LED configuration are missing and the LED Strip stays dark. In `user_config_override.h` I added the necessary `USE_WS2812_HARDWARE NEO_HW_SK6812` and `USE_WS2812_CTYPE NEO_GRBW` defines. I checked if my wiring is faulty, but everything seems to work with WLED flashed._

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): [LilyGO T-OI Plus ESP32-C3](https://de.aliexpress.com/item/1005003348936965.html)
- [x] Tasmota binary firmware version number used: 11.0.0.4 (development)
  - [ ] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: PIO
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
21:47:06.515 RSL: RESULT = {"NAME":"ESP32C3","GPIO":[1,1,1376,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,1,1,1,1],"FLAG":0,"BASE":1}
21:47:06.750 RSL: RESULT = {"Module":{"1":"ESP32C3"}}
21:47:07.001 RSL: RESULT = {"GPIO0":{"0":"None"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"0":"None"},"GPIO4":{"0":"None"},"GPIO5":{"0":"None"},"GPIO6":{"0":"None"},"GPIO7":{"0":"None"},"GPIO8":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO18":{"0":"None"},"GPIO19":{"0":"None"},"GPIO20":{"0":"None"},"GPIO21":{"0":"None"}}
```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
```
- [x] Provide the output of this command: `Status 0`:
```lua
21:48:58.732 RSL: STATUS = {"Status":{"Module":1,"DeviceName":"Tasmota","FriendlyName":["Tasmota"],"Topic":"tasmota_B64950","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
21:48:58.766 RSL: STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota32/release/tasmota32c3.bin","RestartReason":"Software reset digital core","Uptime":"0T00:02:38","StartupUTC":"2022-03-25T20:46:20","Sleep":50,"CfgHolder":4617,"BootCount":4,"BCResetTime":"2022-03-25T21:39:59","SaveCount":10}}
21:48:58.799 RSL: STATUS2 = {"StatusFWR":{"Version":"11.0.0.4(tasmota)","BuildDateTime":"2022-03-25T21:36:20","Core":"2_0_3","SDK":"v.4.4.0","CpuFrequency":160,"Hardware":"ESP32-C3","CR":"399/699"}}
21:48:58.812 RSL: STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["guest",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A190000000000","00000080","00006000","00004000"]}}
21:48:58.840 RSL: STATUS4 = {"StatusMEM":{"ProgramSize":1398,"Free":1856,"Heap":199,"StackLowMark":4,"PsrMax":0,"PsrFree":0,"ProgramFlashSize":4096,"FlashSize":4096,"FlashFrequency":80,"FlashMode":3,"Features":["00000809","9FDAC7CF","0015A001","B7F7BFCF","05DA9BC0","E0360DC7","000840D2","20200000","0034482D"],"Drivers":"1,2,3,4,5,7,8,9,10,11,12,14,16,17,20,21,24,26,27,29,34,35,38,50,52,56,59,86","Sensors":"1,2,3,5,6,7,8,9,10,11,12,13,14,15,17,18,19,20,21,22,26,28,31,34,37,39,40,42,43,45,51,52,55,56,58,59,64,66,67,74,85,92,95,127"}}
21:48:58.884 RSL: STATUS5 = {"StatusNET":{"Hostname":"tasmota-B64950-2384","IPAddress":"192.168.255.143","Gateway":"192.168.255.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.255.1","DNSServer2":"0.0.0.0","Mac":"7C:DF:A1:B6:49:50","Webserver":2,"HTTP_API":1,"WifiConfig":2,"WifiPower":17.0}}
21:48:58.907 RSL: STATUS6 = {"StatusMQT":{"MqttHost":"","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_B64950","MqttUser":"DVES_USER","MqttCount":0,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
21:48:58.931 RSL: STATUS7 = {"StatusTIM":{"UTC":"2022-03-25T20:48:58","Local":"2022-03-25T21:48:58","StartDST":"2022-03-27T02:00:00","EndDST":"2022-10-30T03:00:00","Timezone":"+01:00","Sunrise":"06:42","Sunset":"19:09"}}
21:48:58.957 RSL: STATUS10 = {"StatusSNS":{"Time":"2022-03-25T21:48:58","ESP32":{"Temperature":42.5},"TempUnit":"C"}}
21:48:58.961 RSL: STATUS11 = {"StatusSTS":{"Time":"2022-03-25T21:48:58","Uptime":"0T00:02:38","UptimeSec":158,"Heap":198,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":40},"Wifi":{"AP":1,"SSId":"guest","BSSId":"36:8A:AE:B7:33:5A","Channel":9,"Mode":"11n","RSSI":100,"Signal":-40,"LinkCount":1,"Downtime":"0T00:00:03"}}}
```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
```

### TO REPRODUCE
_Steps to reproduce the behavior:_
1. Compile & Flash Tasmota to the ESP32C3
2. Configure your Template
3. Reboot
4. There is no Slider

### EXPECTED BEHAVIOUR
_After setting up the Template the LED options should be there._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment, marked as answer]
You have not configured any pin(s) for neopixel, and nothing works without appropriate pin definition.

Your template is not activated, if that's what you want to use.

**Reply by TheBattleWolf** [reply]
Activating the template did the trick, tanks for your help :-)
